Documentation/4. Constructors and Destructors/README.md

```diff
@@ -0,0 +1,137 @@
+## <u>Constructors and Destructors</u>
+
+### <u>Constructors</u>
+
+A constructor is a particular member function having the same name as the class name. It calls automatically whenever the object of the class is created.
+
+eg:-
+
+```c++
+#include <iostream.h>  
+#include <conio.h>  
+using namespace std;  
+class hello {     // The class  
+  public:           // Access specifier  
+   hello () {     // Constructor  
+      cout << "Hello World! Program in C++ by using Constructor";  
+    }  
+   void display() {  
+     cout <<"Hello World!" <<endl;  
+   }  
+};  
+int main() {  
+  hello myObj;   /  
+  return 0;  
+}  
+```
+
+<b>There are four types of constructors used in C++.</b>
+
+- Default constructor
+- Dynamic constructor
+- Parameterized constructor
+- Copy constructor
+
+#### Default constructor
+
+A constructor is a class which accepts no parameter and is called a default constructor. If there is no constructor for a class, the compiler implicitly creates a default constructor.
+
+```c++
+class class_name {  
+private:  
+………..  
+………..  
```

Comment:
What is this? Just add a parameter or something here

Reply:
ok